### ISSUE TITLE
Question about impact of drop_condition_rate defaulting to 0 in training

### ISSUE THREAD

**Issue by wyhlovecpp** [PKU-YuanGroup/UniWorld-V1#25]
Hi authors,
Thanks for the great work! I noticed in the training code there's a drop_condition_rate (or drop_prompt_rate) parameter that gets passed into the dataset, and later affects the condition dropout during training. 

My questions:
Do you think setting drop_condition_rate=0 has a negative impact on the model's ability to generalize or perform conditional generation tasks robustly? I noticed that several other models (e.g., InstructPix2Pix, Ultraedit) also use condition dropout during training to improve robustness, always with a non-zero value. Did you try tuning this parameter, and if so, what values work well? Would it make sense to set a non-zero default in the schema to avoid accidental silent behavior?

Thanks!
